### PR TITLE
Increasing poll period for RSS and HTML to 1 hour.

### PR DESCRIPTION
**Pull request by jcjimenez**

### CODE REVIEW

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/HTMLStreamFactory.scala [hunk 1]
```diff
@@ -26,7 +26,7 @@ class HTMLStreamFactory extends StreamFactoryBase[HTMLPage] with Loggable {
             "User-Agent" -> params.getOrElse("userAgent", "Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36").toString
           ),
           maxDepth = params.getOrElse("maxDepth", "1").toString.toInt,
-          pollingPeriodInSeconds = params.getOrElse("pollingPeriodInSeconds", "60").toString.toInt,
+          pollingPeriodInSeconds = params.getOrElse("pollingPeriodInSeconds", "3600").toString.toInt,
```
Comment (jcjimenez) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/HTMLStreamFactory.scala:29]: 60s -> 1hr

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/RSSStreamFactory.scala [hunk 2]
```diff
@@ -26,7 +26,7 @@ class RSSStreamFactory extends StreamFactoryBase[RSSEntry] with Loggable {
           ),
           connectTimeout = params.getOrElse("connectTimeout", "1000").toString.toInt,
           readTimeout = params.getOrElse("readTimeout", "3000").toString.toInt,
-          pollingPeriodInSeconds = params.getOrElse("pollingPeriodInSeconds", "10").toString.toInt,
+          pollingPeriodInSeconds = params.getOrElse("pollingPeriodInSeconds", "3600").toString.toInt,
```
Comment (jcjimenez) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/RSSStreamFactory.scala:29]: 10s -> 1hr

**Review by Smarker — APPROVED** [review]
LGTM

**Review by erikschlegel — COMMENTED** [review]
LGTM

**Pull request merged**